web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -250,8 +254,23 @@ function ClusterFilterItem(props: SearchResultItem<SearchResultCluster>) {
 function ResourceTypeFilterItem(
   props: SearchResultItem<SearchResultResourceType>
 ) {
+  const resourceIcons: Record<
```

Comment:
Could you move this outside of the component body? 😏

Reply:
I finally got around to looking this up and there's a reply from a V8 dev here: https://stackoverflow.com/questions/58432310/do-javascript-engines-optimize-constants-defined-within-closures

Reply:
Thanks for this, now we have scientific proof :)